app/apis/crypto_exchanges/binance_api.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module CryptoExchanges
+  # Binance API
+  # https://github.com/binance-us/binance-official-api-docs/blob/master/rest-api.md#general-api-information
+  class BinanceApi < CryptoExchanges::BaseApi
+    BASE_URL = 'https://api.binance.us/api'
+
+    def symbol
```

Comment:
for all of the crypto exchanges, `symbol` can be brought down to the private section. `orderbook` appears to be your only public method so far.